drush-services.yml

```diff
@@ -0,0 +1,25 @@
+parameters:
```

Comment:
No, this file is for Drush core only. I agree that comments here would be helpful anyway.

My idea is that Drush could also look for a services file at the document root; this would allow Backdrop & c. to add their own dependency injection easily. We'd load the Drush services first, which would allow others to refer to labels like 'logger' and 'bootstrap.manager' without needing to know exactly what their implementation was, so long as they fulfilled the contract. As I mentioned in a comment, I am going to re-implement the way Boot classes are added to the BootstrapManager for better extensiblity.

It might also make sense for Drush to look for service files anywhere that it currently looks for Drush extensions. This will need a little consideration and discussion, but I think that ultimately this could result in the ability to write extensions that are very separable / testable.